### ISSUE TITLE
Cannot save required field with value `0`

### ISSUE THREAD

**Issue by RaschidJFR** [parse-community/parse-dashboard#1980]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

Getting error `Please enter all required fields` when setting value `0` in a required column

### Steps to reproduce

1. Create a new class
2. Add a column "MyColumn" of type `Number` and mark it as `required`
3. Add a new row and with a value of `0` in "MyColumn"

### Actual Outcome

Getting error: `Please enter all required fields`

![image](https://user-images.githubusercontent.com/8172495/147969903-54664e17-5034-4a2c-809a-9c1b22ddec2d.png)

### Expected Outcome

`0` should be accepted as a valid value for "MyColumn"

### Environment

Dashboard
- Parse Dashboard version: `3.3.0`
- Browser (Safari, Chrome, Firefox, Edge, etc.): `Chrome`
- Browser version: `96.0.4664.110`

Server
- Parse Server version: `4.10.4`
- Operating system: `MacOS 11.3.4`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local`

Database
- System (MongoDB or Postgres): `MongoDb`
- Database version: `3.6.8`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `local`

**Comment by RaschidJFR** [comment]
I think the problem is in [Browser.react.js#L360](https://github.com/sadakchap/parse-dashboard/blob/cb7d80b91156381b980bde0696b65c9f90b61b5d/src/dashboard/Data/Browser/Browser.react.js#L360).

```js
// Browser.react.js
// ...
    if (requiredCols.length) {
      for (let idx = 0; idx < requiredCols.length; idx++) {
        const name = requiredCols[idx];
        if (!obj.get(name)) {
          this.showNote("Please enter all required fields", true);
          this.setState({
            markRequiredField: true
          });
          return;
        }
      }
    }
// ...
```

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.0-alpha.9](https://github.com/parse-community/parse-dashboard/releases/tag/4.0.0-alpha.9)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.0-beta.3](https://github.com/parse-community/parse-dashboard/releases/tag/4.0.0-beta.3)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [4.0.1](https://github.com/parse-community/parse-dashboard/releases/tag/4.0.1)
